_includes/tools/jspm/usage.md

```diff
@@ -0,0 +1,30 @@
+Create a new project configuration (i.e. config.js):
+
+```sh
+$ jspm init
+```
+when asked which transpiler you would like to use:
+
+> Which ES6 transpiler would you like to use, Traceur or Babel? [traceur]:
+
+Type Babel and press enter.
```

Comment:
Babel => \`babel\` (with backticks) to accurately reflect what users actually type.